### DISCUSSION TITLE
When are dockets updated?

### DISCUSSION THREAD

**Discussion by mlissner** [freelawproject/courtlistener#1520] [Q&A]
When I look at a page, it says that the case was updated on December 23rd, but the latest docket entry is from before then. When do you get new docket entries? Can you explain what's going on here?

**Comment by mlissner** [comment]
Many of our dockets webpages contain information that has to be purchased from a federal or state judiciary. Each of these cases costs money to keep it up to date, and there are hundreds of thousands of such cases. Very few organizations have the money to do this for so many cases, and so most systems have some way of shifting that cost to the users so that one way or another the users are the ones paying for the content. 

This is an elegant solution because the most interesting content is the most likely to be paid for by users, but it also means that sometimes *you're* the person that's most interested in getting an update. When this happens, *you* are the one that needs to buy it and supply it to everybody.

The way we do this is via the [RECAP Extensions](https://free.law/recap/) for Chrome and Firefox. You install the extension and then anything you buy from PACER gets sent to us to be shared via CourtListener. Every day, we get thousands of documents this way.

When we *don't* get content this way for a case, we use whatever other methods we can think of to get free content from the judiciary. For PACER, there are [numerous ways we use](https://www.courtlistener.com/coverage/), but sometimes all we can learn about a case is "was it updated," not the content of the actual update. 

So, to sum up, dockets are updated when:
 - We can get free content from the judiciary and do so; or
 - We get paid content via crowdsourcing tools like RECAP.

In some cases, that means we know a case has been updated, but we lack the update. In that case, if you want the new content, it may be up to you to go get it and share it with the world yourself.
